concent_api/core/decorators.py

```diff
@@ -167,6 +168,11 @@ def wrapper(request: HttpRequest, *args: Any, **kwargs: Any) -> Union[HttpRespon
                     },
                     status=400
                 )
+            except NonPositivePriceTaskToComputeError as exception:
```

Comment:
please write a simple Django test for this case

Reply:
That simple test, cause major refactor of returning ServiceRefused :P I guess our previous implementation was broken. But it's done now.